### PR TITLE
🚀 feat: Added author modal as per figma and updated Chapter Header

### PR DESCRIPTION
**Pull request by Aniket-508**

### CODE REVIEW

**Comment by Aniket-508** [comment]
@Gupta-Anubhav12 @akshitarora921 Please have a look at this!

File: components/Headers/PageHeader.js [hunk 1]
```diff
@@ -98,13 +98,13 @@ const PageHeader = () => {
                       </a>
                     </Link>
 
-                    <Link href="/verseParellel">
+                    <Link href="/verseParallel">
```
Comment (akshitarora921) [inline components/Headers/PageHeader.js]: use `verse-parallel` as discussed and update the file name also.

File: components/Headers/ChapterHeader.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
+import { useState, useEffect } from "react";
```
Comment (akshitarora921) [inline components/Headers/ChapterHeader.js:1]: Great work 🚀 , but we can achieve this using `PageHeader` also as we need to hide 2 options for the chapter page only. @Gupta-Anubhav12 Any thoughts?

File: components/Shared/Author.js [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+                      pointer-events-none inline-block h-[25px] w-[25px] rounded-full bg-white shadow-lg transform ring-0 transition ease-in-out duration-200`}
+                    />
+                  </Switch>
+                </div>
+
+                <div
+                  class="mb-6"
+                  hidden={!isVerseTransliterationLanguageEnabled}
+                >
+                  <Listbox
```
Comment (akshitarora921) [inline components/Shared/Author.js:125]: I am not able to see `ListBox` on the website. Please check.
image

File: layouts/ChapterLayout.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (akshitarora921) [inline layouts/ChapterLayout.js:1]: Same for this we can use ` PageLayout`

**Review by akshitarora921 — CHANGES_REQUESTED** [review]
@Aniket-508 Great work. I think you over-did a few things but keep up the good work 🎖️ . Thanks for the contribution 💯

File: components/Shared/Author.js (hunk 3 above)
Comment (Aniket-508) [reply components/Shared/Author.js:125]: You get to see that whenever toggle is on!

Comment (akshitarora921) [reply components/Shared/Author.js:125]: oh, ok. There is an issue in the dark mode.
image

**Comment by Aniket-508** [comment]
@akshitarora921 @Gupta-Anubhav12 Please check it once now!

File: components/Headers/ChapterHeader.js [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+                        >
+                          <img className="w-6 h-6" src="/Home.svg" />
+                          Home
+                        </a>
+                      </Link>
+                    </div>
+                    <Link href="/">
+                      <a
+                        href="#"
+                        className="border-transparent text-current flex flex-col items-center p-2 rounded border-b-2 text-sm font-medium hover:bg-nav-hover dark:hover:bg-dark-bg"
```
Comment (akshitarora921) [inline components/Headers/ChapterHeader.js:68]: add onClick

**Comment by Aniket-508** [comment]
@akshitarora921 Have a look once and merge please!

**Review by akshitarora921 — APPROVED** [review]
LGTM!

**Pull request merged**